helix-core/src/main/java/org/apache/helix/manager/zk/ZKHelixAdmin.java

```diff
@@ -1025,6 +1026,25 @@ public void dropResource(String clusterName, String resourceName) {
     accessor.removeProperty(keyBuilder.resourceConfig(resourceName));
   }
 
+  @Override
+  public void addCloudConfig(String clusterName, CloudConfig cloudConfig) {
+    logger.info("Add CloudConfig to cluster {}, CloudEnabled {}, CloudID {}, CloudProvider{}",
```

Comment:
We should log the whole config instead of only 3 fields.

Reply:
Fixed.